open_spiel/games/phantom_go_test.cc

```diff
@@ -0,0 +1,198 @@
+    std::shared_ptr<const Game> game =
+        LoadGame("phantom_go", params);
+    PhantomGoState state(game, kBoardSize, kKomi, 0);
+
+    for (int i = 0; i < 150; i++) {
+        std::vector<Action> actions = state.LegalActions();
+        std::shuffle(actions.begin(), actions.end(), std::mt19937(std::random_device()()));
+        state.ApplyAction(actions[0]);
+        if (state.IsTerminal()) {
+            break;
+        }
+    }
+
+    std::unique_ptr<State> resampleState = state.ResampleFromMetaposition(0, nullptr);
+
+    std::cout << "Original state\n" << state.ToString();
+
+    std::cout << "Resampled state\n " << resampleState->ToString();
+
+    /*for(int i = 0; i < state.FullHistory().size(); i++)
+    {
+        std::cout << state.ActionToString(state.FullHistory()[i].player, state.FullHistory()[i].action) << " " <<
+            state.ActionToString(resampleState->FullHistory()[i].player, resampleState->FullHistory()[i].action) << "\n";
+    }*/
+}
+
+//This test was used to test metaposition resampling on large ammounts of states
+//  with different lengths
+void ResampleFromInfostateForceTest() {
+    std::cout << "Starting ResampleFromMetaposition visual Test\n";
+    GameParameters params;
+    params["board_size"] = GameParameter(kBoardSize);
+    /*std::shared_ptr<const Game> game =
+        LoadGame("phantom_go", params);
+    PhantomGoState state(game, kBoardSize, kKomi, 0);*/
+
+    for (int n = 10; n < 20; n++) {
+        std::cout << "Starting test for n " << n << "\n";
+        for (int x = 0; x < 2000; x++) {
+            std::shared_ptr<const Game> game =
+                LoadGame("phantom_go", params);
+            PhantomGoState state(game, kBoardSize, kKomi, 0);
+
+            for (int i = 0; i < n * 10; i++) {
+                if (state.IsTerminal()) {
+                    state.UndoAction(-1, -1);
+                    break;
+                }
+                std::vector<Action> actions = state.LegalActions();
+                std::shuffle(actions.begin(), actions.end(), std::mt19937(std::random_device()()));
+                for (long action: actions) {
+
+                    if (action != VirtualActionToAction(kVirtualPass, kBoardSize)) {
+                        state.ApplyAction(action);
+                        break;
+                    }
+                }
+
+            }
+            std::unique_ptr<State> resampleState = state.ResampleFromMetaposition(state.CurrentPlayer(), nullptr);
+
+        }
+    }
+}
+
+}  // namespace
+}  // namespace phantom_go
+}  // namespace open_spiel
+
+int main(int argc, char **argv) {
+    open_spiel::phantom_go::CloneTest();
+    open_spiel::phantom_go::BasicGoTests();
+    open_spiel::phantom_go::HandicapTest();
+    open_spiel::phantom_go::ConcreteActionsAreUsedInTheAPI();
+    open_spiel::phantom_go::IllegalMoveTest();
+    open_spiel::phantom_go::StoneCountTest();
+    //open_spiel::phantom_go::ResampleFromInfostateVisualTest();
```

Comment:
Enable tests

Reply:
First, I have removed the Visual Test, since it is not an actual test. Second, I think the Force test should be disabled by default, because it is designed to test the resampling on a large number of simulations, and can take several minutes to complete.